core/tests/protractor_desktop/ExplorationTranslationTab.js

```diff
@@ -131,11 +131,71 @@ describe('Exploration translation', function() {
     explorationEditorPage.saveChanges();
 
     explorationEditorPage.navigateToTranslationTab();
+    explorationEditorTranslationTab.exitTutorial();
     explorationEditorTranslationTab.expectNumericalStatusToMatch(
       '(0/8)');
+
+    // click upload button
+    explorationEditorTranslationTab.openUploadAudioModal();
+    var audioToUpload = '../data/cafe.mp3';
+    var audioAbsolutePath = path.resolve(__dirname, audioToUpload);
+    var audioElem = element(by.css('[ng-class="inputFieldClassName"]'));
+    audioElem.sendKeys(audioAbsolutePath);
+    explorationEditorTranslationTab.saveUploadedAudio();
+    explorationEditorTranslationTab.expectNumericalStatusToMatch(
+      '(1/8)');
     users.logout();
   });
 
+  it('should provide correct status color ' +
+    'for each card(state) in Exploration Overview', function() {
```

Comment:
Maybe you can have `should provide correct status color for each state in the graph view`.

Reply:
done